include/runProcess.h

```diff
@@ -21,6 +21,8 @@
 
 #include <unistd.h>
 #include <sys/types.h>
+#include <pwd.h>
```

Comment:
I think this should be conditional on the OS _not_ being Windows, IIUC that's where the AppVeyor failure is coming from.

Reply:
Strange how `unistd.h` is available on Windows yet `pwd.h` is not. Regardless, this is now properly guarded.